public/index.html

```diff
@@ -30,15 +30,13 @@
   src="https://code.jquery.com/jquery-3.5.1.min.js"
   integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
   crossorigin="anonymous"></script>
-  <script src="https://ultimate-nike.s3-us-west-1.amazonaws.com/bundle-photo-module.js"></script>
-    <!-- product header -->
-  <script src="https://simplicity-product-service.s3-us-west-2.amazonaws.com/product-bundle.js"></script>
+  <!-- <script src="https://ultimate-nike.s3-us-west-1.amazonaws.com/bundle-photo-module.js"></script> -->
+  <!-- product header -->
+  <!-- <script src="https://simplicity-product-service.s3-us-west-2.amazonaws.com/product-bundle.js"></script> -->
   <!-- inventory -->
-  <!-- <script src="https://ultimate-nike-inventory.s3.us-east-2.amazonaws.com/bundle.js"></script> -->
-  <script src="http://3.137.143.152:3004/bundle.js"></script>
-    <!-- reviews -->
-  <!-- <script src="https://simplicity-reviews.s3-us-west-1.amazonaws.com/bundle.js"></script> -->
-  <script src="http://18.219.146.205:3003/bundle.js"></script>
+  <script src="http://localhost:8000/bundle.js"></script>
+  <!-- reviews -->
+  <!-- <script src="http://18.219.146.205:3003/bundle.js"></script> -->
```

Comment:
Looks good. Agree with commenting out instead of deleting as will be reimplemented.